packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -16,6 +16,14 @@ export 'package:flutter/rendering.dart' show
   SliverGridDelegateWithFixedCrossAxisCount,
   SliverGridDelegateWithMaxCrossAxisExtent;
 
+
+/// A callback which produces a semantic index given a widget and the local index.
+///
+/// Return a null value to prevent a widget from receiving an index.
```

Comment:
This should probably explain what a semantic index is (or link to an explanation).